linkerd2-proxy/Cargo.toml

```diff
@@ -12,6 +12,7 @@ default = ["multicore", "meshtls-rustls"]
 multicore = ["tokio/rt-multi-thread", "num_cpus"]
 meshtls-boring = ["linkerd-meshtls/boring"]
 meshtls-rustls = ["linkerd-meshtls/rustls"]
+meshtls-fips = ["linkerd-meshtls/fips"]
```

Comment:
sorry, could we call this `meshtls/boring-fips` (and `linkerd-meshtls/boring-fips`)?